scripts/install/ceres_install.bash

```diff
@@ -20,24 +21,30 @@ install_suitesparse_fix() {
 }
 
 install_ceres_solver() {
-    # clone ceres solver
     mkdir -p $SRC_PREFIX_PATH
     cd $SRC_PREFIX_PATH
-    git clone https://github.com/ceres-solver/ceres-solver.git
+
+    # clone ceres-solver if directory does not already exist, or pull
+    if [ ! -d ceres-solver ]; then
+       git clone https://github.com/ceres-solver/ceres-solver.git
+    else
+       git -C ceres-solver pull
```

Comment:
This line won't work the second time as you check out a tag version(you are not in any branch) later on.